test/BasicWorldWindowController.test.js

```diff
@@ -54,6 +54,7 @@ define([
             return {left: 339.5, top: 225};
         }
     };
+    wwd.layers = [];
```

Comment:
Would you please clarify why this change is needed in the tests?

Reply:
@pdavidc I added this to fix a unit test breakage when we converted ScreenCreditController to a layer.

Reply:
Also, I moved the Mock WWD creation code to a common routine in the camera branch, so these three changes will all be consolidated.

Reply:
Thanks for clarifying, Mark.